src/donationswap.py

```diff
@@ -1085,21 +1089,24 @@ def _send_mail_about_match(self, my_offer, their_offer, match_secret):
 			their_offer.country.currency.iso,
 			my_offer.country.currency.iso)
 
+		myGiftAidMultiplier = (my_offer.country.gift_aid / 100.0) + 1
```

Comment:
actually... how about you add this at line 349 of `entities.py`:

```	@property
	def gift_aid_multiplier(self):
		return 1/ (1 + self.country.gift_aid / 100)

```

Reply:
you have good ideas :+1: